docs/ml/vision/mlmodel.md

```diff
@@ -117,6 +117,18 @@ Add the vision service object to the services array in your raw JSON configurati
 
 Click **Save config**.
 
+The following attributes are available for an `mlmodel` detector or classifier:
+
+<!-- prettier-ignore -->
+| Parameter | Type | Inclusion | Description |
+| --------- | ---- | --------- | ----------- |
+| `mlmodel_name` | string | **Required** | The name of the [ML model service](/ml/deploy) you want to use the model from. |
+| `remap_output_names` | object | Optional | The names of your output tensors, mapped to the service requirements. See [Tensor names](#tensor-names) for more information. |
+| `remap_input_names` | object | Optional | The name of your input tensor, mapped to the service requirements. See [Tensor names](#tensor-names) for more information. |
+| `input_image_bgr` | bool | Optional | Set this to `true` if the ML model service expect the input image to have BGR pixels, rather than RGB pixels. |
+| `input_image_mean_value` | array | Optional | The standard deviation of the RGB (or BGR) values. Only required if the ML model service expects the input image to be normalized. |
+| `input_image_std_dev` | array | Optional | The standard deviation of the RGB (or BGR) values. Only required if the ML model service expects the input image to be normalized. |
```

Comment:
@bhaney questions here: in jira you phrased as "ML model expects" so just want to confirm it's ML model service and not the vision service?
- How would a user know if the ML model expects the input image to be normalized (or have BGR pixels)?
- What are the defaults for input_image_mean_value and input_image_std_dev? I couldn’t find in code. Also, input_image_bgr default I'm assuming is true?

Reply:
- Correct, it's the ML Model service that is expecting these things
- Unfortunately, you just have to know about the model you've chosen. Models are not really standardized - people create the inputs and outputs based on their whims. Places like google and meta have standard, which we have baked into the default behavior of the vision service, but if you are using an ML Model service that loads a model you got from a random person online, or that you built yourself, only the user will know if they need these options or not.
- correct, input_image_bgr is false by default, meaning is assumes images come in an RGB format. the default forn mean_value is [0.5, 0.5, 0.5] and the default for std_dev is also [0.5, 0.5, 0.5].

Reply:
@bhaney perfect, thanks!